files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -860,7 +860,34 @@ protected function debug( $message ) {
 			return;
 		}
 
-		// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
-		error_log( sprintf( '%s (%s)', $message, wp_debug_backtrace_summary() ) );
+		$trace = $this->backtrace_fmt();
+
+		\Automattic\VIP\Logstash\log2logstash(
+			[
+				'severity' => 'info',
+				'feature'  => 'stream_wrapper_audit_' . $trace[1]['function'],
+				'message'  => "File op {$trace[1]['function']}: " . $message,
+				'extra'    => [
+					'trace' => $this->backtrace_fmt(),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Format the debug backtrace to be a bit more readable .
+	 *
+	 * @return array
+	 */
+	private function backtrace_fmt() {
+		$t = debug_backtrace( false, 30 ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_debug_backtrace
```

Comment:
I think a more descriptive variable name here would be helpful.

Reply:
t for trace :)